web-console/future-openshift/bulk-actions/bulk-actions.md

```diff
@@ -0,0 +1,26 @@
+# Bulk actions
+
+Bulk actions will be added to a number of pages in the OpenShift console.
+
+
+## Behavior
+* The bulk selector will be the first component in the toolbar on pages where bulk actions are available.
+* Users can click the checkbox in the bulk selector to select all items in the list. If all items are selected, the bulk selector checkbox will be filled with a check and the bulk selector will read 'All items selected'.
+* Users can also choose to select a group of items by clicking the associated checkboxes next to them. The bulk selector checkbox will be filled in with a '-' to indicate that some items are selected, and the bulk selector will display the number of items selected (e.g. '2 selected').
+* Clicking the bulk selector checkbox will deselect all items.
+* If any of the items in the list are selected, the primary action button will be replaced by the actions dropdown.
+
+![1](img/1.png)
```

Comment:
@bmignano I think we need to update this image since we decided to move the create button elsewhere. Does this change the actions behavior too so now that button will always be visible? Not sure if I am remembering correctly. Let me know if these are the latest designs based on the feedback from Sam and company in the stakeholders meeting!